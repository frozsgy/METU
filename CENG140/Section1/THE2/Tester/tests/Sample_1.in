ADD SH4RPSH00T3R11
ADD MARKHAMILL<3<3
ADD IFollowAllTopics
ADD IFollowNoTopic
ADD IWillGetDeleted:(
ADD HowToUnsubscribe
SUBSCRIBE SH4RPSH00T3R11 Basketball
SUBSCRIBE MARKHAMILL<3<3 StarWars
SUBSCRIBE IFollowAllTopics Basketball
SUBSCRIBE IFollowAllTopics StarWars
SUBSCRIBE HowToUnsubscribe Basketball
SEND IFollowNoTopic StarWars
SEND IFollowAllTopics Basketball
SEND IWillGetDeleted:( StarWars
DELETEUSER IWillGetDeleted:(
ADD YetAnotherUser
UNSUBSCRIBE HowToUnsubscribe Basketball
SUBSCRIBE HowToUnsubscribe StarWars
SEND MARKHAMILL<3<3 StarWars
SEND MARKHAMILL<3<3 Basketball
