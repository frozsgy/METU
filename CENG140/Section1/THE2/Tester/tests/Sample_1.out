1: 2 5
2: 1 4 5
3: 1 2 4 5
4: 1
6: 4
7:
